md/custom-widgets.md

```diff
@@ -21,6 +21,12 @@ For more information, see the AngularJS guide on [templates](https://docs.angula
 
 For now, the definition of custom containers is not yet available in the product.
 
+::: warning
+Note that UI Designer always prefix the ID of the custom widget by "custom", 
+whereas the [standard widgets](widgets.md) are internally using the prefix "pb".
+If you build custom widgets outside of the UI Designer, you must not use "pb" as widget ID prefix.
```

Comment:
rather
", your widget ID must not start with "pb"."

Reply:
done